frontend/src/util/worksheet_utils.js

```diff
@@ -7,6 +7,10 @@ export function renderDuration(s) {
     // s: number of seconds
     // Return a human-readable string.
     // Example: 100 => "1m40s", 10000 => "2h46m"
+    if (typeof s === 'undefined') {
+        return '';
+    }
+
     if (s === null) {
```

Comment:
These two cases shold be merged - `if (s == null)` should work.  Make the return `<none>`.